### PR TITLE
Water and death stop burning

### PR DESCRIPTION
**Pull request by olliestanley**

### CODE REVIEW

**Comment by natemort** [comment]
Why hasn't this been pulled?

File: src/main/java/org/spout/vanilla/component/entity/misc/Burn.java [hunk 1]
```diff
@@ -64,10 +64,12 @@ public void onTick(float dt) {
 			} else {
 				rainTimer = 0f;
 			}
-			if (rainTimer >= 2.0f) {
+			if (rainTimer >= 2.0f || point.getBlock().getMaterial() == VanillaMaterials.WATER || health.isDead()) {
```
Comment (Afforess) [inline src/main/java/org/spout/vanilla/component/entity/misc/Burn.java:67]: == Water is bad. It doesn't cover stationary water. Use instanceof Water instead.

**Comment by Afforess** [comment]
Probably because there are problems with it.

File: src/main/java/org/spout/vanilla/component/entity/misc/Burn.java (hunk 1 above)
Comment (olliestanley) [inline src/main/java/org/spout/vanilla/component/entity/misc/Burn.java:67]: Shall do, it should be done today hopefully

**Comment by Sleaker** [comment]
re-implemented a bit different, thanks @DziNeIT for the attempt.

**Pull request closed without merging**